go/lambda-api-ci/cmd/lambda/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"github.com/aws-samples/aws-cdk-examples/go/lambda-api-ci/cmd/internal/constants"
+	"github.com/aws-samples/aws-cdk-examples/go/lambda-api-ci/lib/lambdaapistack"
+	"github.com/aws/aws-cdk-go/awscdk"
+	"github.com/aws/jsii-runtime-go"
+)
+
+func main() {
+	defer jsii.Close()
```

Comment:
?!!?

Reply:
You gotta be explicit, and if you don't do that, the runtime won't "clean itself up" (i.e: temporary directories)